pkg/operator/controllers/guardrails/policies/README.md

```diff
@@ -212,25 +221,54 @@ spec:
           cpu: "100m"
           memory: "30Mi"
 ```
-the assertions section is the expected result
+the `assertions` section is the expected result
 
-gator test is done via cmd:
+gator test can be done via cmd:
```

Comment:
Should Gator be capitalized here? Noticed you fixed that in a few places but not here.

Reply:
Aha, good catch, as [the official website](https://open-policy-agent.github.io/gatekeeper/website/docs/gator/) is using lower case all the time, I'll follow that as well to make it more consistent

Reply:
change made, thank you for the review